src/editpage/components/blocks/render.js

```diff
@@ -0,0 +1,19 @@
+const md = require('markdown-it')();
+const mk = require('@iktakahiro/markdown-it-katex');
+md.use(mk);
+
+// eslint-disable-next-line valid-jsdoc
```

Comment:
Why disable the warning but still have the jsdoc?

Reply:
more documentation is good, but I did not understand the error. Any idea what it could've been?

Reply:
Probably the `{}` in returns or maybe it wants `return` instead of `returns` something like that usually.